HomeWorks/HW04.Task1/Sum_result.cs

```diff
@@ -0,0 +1,47 @@
+﻿using  System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HW04.Task1
+{
+    class Sum_result
```

Comment:
do not use `_` in classes, methods, variables, etc. 

just only one thing where it can be used - private fields